nbuild.py

```diff
@@ -71,8 +84,9 @@ def main():
     os.environ['PATH'] = '/bin:/sbin/:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/opt/bin'
     os.environ['MAKEFLAGS'] = f'-j{cpu_count() + 1}'
 
-    # Override those with the ones within the .env file.
-    load_dotenv(dotenv_path='.env', override=True)
+    # Override environment with the content of the config file
+    if core.config.get_config()['env'] is not None:
```

Comment:
Is it still safe to use if the `env` section is not provided in the config file ? I think dictionaries raise exceptions when a key cannot be found

Reply:
Nop, you're right, i mixed up with `.get()`.